src/Relude/Lifted/Terminal.hs

```diff
@@ -23,8 +23,11 @@ import Relude.String.Reexport (String, Text)
 import qualified Data.Text.IO as TIO
 import qualified System.IO as IO (print, putStr, putStrLn)
 
+{- | Lifted version of 'Data.Text.getLine'.
 
--- | Lifted version of 'Data.Text.getLine'.
+TODO: getLine
```

Comment:
I'm not sure that doctest for `getLine` function would work :slightly_smiling_face: